index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav>
+      <a href=""><img src="images/Logo Placeholder.svg" alt="" /></a>
+
+      <a href=""><img src="images/Icon - Menu.svg" alt="" /></a>
+    </nav>
+    <header class="header-section">
+      <div>
+        <h1>
+          Hey there. I’m Joseph <br />
+          <span class="header-span"> I’m a software developer</span>
+        </h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="social">
```

Comment:
- [ ] The webpage is not displaying the social icons. Kindly make sure everything is implemented correctly.

## Your Design:
![image](https://user-images.githubusercontent.com/92215878/204805054-68542a84-fde5-4388-918c-9ded6bf38c4a.png)

## Figma Design:
![image](https://user-images.githubusercontent.com/92215878/204805115-55bceda0-f19c-441f-bd23-be5c487beaa8.png)